.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
```

Comment:
nit: extra empty line

Reply:
Had used the github gui to init so the emptylines came with the template I suspect. removed empty line